web34ever/chains.json

```diff
@@ -37,6 +37,87 @@
         "run_time": "10:10",
         "minimum_reward": 1000000
       }
-    }
+    },
+    {
+      "name": "bitcanna",
```

Comment:
Heads up you have a duplicate here, but I'll merge as it shouldn't cause any issues